ui/DesignSystem/Component/Stats.svelte

```diff
@@ -1,13 +1,19 @@
 <script>
   import { Icon, Numeric } from "../Primitive";
 
-  export let stats = [
-    { icon: Icon.Commit, count: -1 },
-    { icon: Icon.Branch, count: -1 },
-    { icon: Icon.Member, count: -1 },
-  ];
+  export let stats = {
+    contributors: -1,
```

Comment:
Why is `-1` the default here? :thinking: Just to differentiate it from a project with a genuine `0` is it?

Reply:
I believe that's the reasoning. I think `null` might make more sense for error-handling.